packages/imperative/src/config/cmd/update-schemas/update-schemas.definition.ts

```diff
@@ -0,0 +1,31 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { join } from "path";
+import { ICommandDefinition } from "../../../../../cmd";
+
+export const updateSchemasDefinition: ICommandDefinition = {
+    name: "update-schemas",
+    aliases: ["us"],
+    type: "command",
+    summary: "update schema files",
+    description: "Update schema files by looking up the directory structure.",
```

Comment:
While we plan to automatically call this when installing a plugin, if we want customers to run this command directly, we may need a more complete description of what they would accomplish with this command. I don't know if we should try to explain some of the edge cases. It might be too wordy and confusing,